internal/pkg/template/template.go

```diff
@@ -0,0 +1,101 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// Package template renders the static files under the "/templates/" directory.
+package template
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/aws/amazon-ecs-cli-v2/templates"
```

Comment:
Do we still need this package? A bit confusing to me to have `template` and `templates`. What do you think if we combine those two packages?

Reply:
Good question!  I'm not sure if we can combine the two 🤔 because the existing `templates` package is used by packr to embed the files into our binary. 
However, the `templates` package is only going to be accessed from this new package so I don't think clients will be confused.